ci-operator/config/openshift/library/openshift-library-master.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: python
+    namespace: openshift
+    tag: "3.6"
```

Comment:
If preferred, the python 3.6 tag can be added to the release imagestream

Reply:
@stevekuznetsov it appears that the rehearse jobs are using a go build (which fails). How can I configure ci-operator such that a go build does not happen?

Reply:
I believe it's not a problem with a go build, ci-operator (or actually, clonerefs inside) fails to create a destination dir for the clone:

```
{"component":"clonerefs","level":"info","msg":"Cloning refs","refs":{"org":"openshift","repo":"library","base_ref":"master","base_sha":"47296d0632a8f95a6f5dc341ab4ab22a92fc0d7b"},"time":"2019-05-23T13:13:20Z"}
{"command":"mkdir -p /go/src/github.com/openshift/library","component":"clonerefs","error":"exit status 1","level":"info","msg":"Ran command","output":"mkdir: cannot create directory ‘/go’: Permission denied\n","time":"2019-05-23T13:13:20Z"}
```

Reply:
@petr-muller yes...but it's creating the destination dir as if it was a golang build.
`openshift/library` uses python to run the verification script, and there is no go binary to build - hence why I'm using the python imagestream as the build root.

Reply:
You'll need to make sure `/go` exists in the image -- there's a way to configure this to not happen but it's less of an effort to just make the dir exist. We need _somewhere_ to put the code